csrc/ir/internal_nodes.h

```diff
@@ -2865,12 +2865,27 @@ class ArgsortOp : public Expr {
 //! This operation performs a grouped matrix multiplication.
 //!
 //! Parameters:
-//! - out: output tensor
+//! - out_mat: output tensor
+//! - out_scale: output block scaling factor tensor
+//! - out_gamma: output global scaling factor tensor
```

Comment:
Interesting, matrix1 and matrix2 don't come with a gamma?

Reply:
matrix1 and matrix2 shares a single alpha.

Because scalar scaling factor on matrix1 and matrxi2 can be combined.

i.e. `(alpha1 * matrix1) @ (alpha2 * matrix2) == alpha * (matrix1 @ matrix2)` where `alpha = alpha1 * alpha2`

Reply:
Got it. Input `gamma`s are folded into `alpha`.